crates/ruff_linter/src/codes.rs

```diff
@@ -1031,6 +1031,7 @@ pub fn code_to_rule(linter: Linter, code: &str) -> Option<(RuleGroup, Rule)> {
 
         // airflow
         (Airflow, "001") => (RuleGroup::Stable, rules::airflow::rules::AirflowVariableNameTaskIdMismatch),
+        (Airflow, "302") => (RuleGroup::Preview, rules::airflow::rules::AirflowDeprecatedMembers),
```

Comment:
Can you tell me a bit about your intended rule numbering? It seems you've something in mind because you chose 302 over e.g 301.

Somewhat related. The numpy rule is called `Numpy2Deprecation`. Should we align this rule's name to `Airflow3Deprecation`?

Reply:
Well simply because I implemented #14581 (which used AIR301) before this. We don’t really have a rule on these; I can switch them around if it is nicer.

(Now that I think of it, should we start with 300 instead…?)

Reply:
> these; I can switch them around if it is nicer.

Ah no, it's fine. Either way works for us.

> (Now that I think of it, should we start with 300 instead…?)

We haven't been very consistent about whether we start with 0 or 1 across linters, but I think we've been mostly consistent for a single inter. That's why I would prefer starting with 301, considering that there's already 001.